src/components/appRouter.js

```diff
@@ -577,7 +577,8 @@ define(['loading', 'globalize', 'events', 'viewManager', 'layoutManager', 'skinM
 
     function showDirect(path) {
         return new Promise(function(resolve, reject) {
-            resolveOnNextShow = resolve, page.show(baseUrl()+path);
+            resolveOnNextShow = resolve;
+            resolve = page.show(baseUrl() + path);
```

Comment:
This doesn’t seem to match the function of the original.

Reply:
It's what I found through research :/ I honestly never use comma operators because I find them insanely dirty, so I'm not sure how to translate this to "proper" code.

What would you use ?

Reply:
a1132e9 should be better.

It's a damn confusing operator.

Reply:
It is really weird. I’m sure that’s a holdover from minification. I’ve never seen anyone actually use it in practice lol

Reply:
There's a few big these dotted around the source, so I'd bet on leftover minification as well.